cli/cli_addr.c

```diff
@@ -224,6 +271,15 @@ static int cmd_off2gen(struct nvm_cli *cli)
 {
 	struct nvm_cli_cmd_args *args = &cli->args;
 
+    if (cli->opts.terse) {
+        for (int i = 0; i < args->ndec_vals; ++i) {
+            struct nvm_addr gen = { 0 };
+            gen = nvm_addr_off2gen(args->dev, args->dec_vals[i]);
+            nvm_addr_terse_prn(&gen, 1, args->dev);
```

Comment:
Here you can just call nvm_addr_pr, since nvm_addr_pr is already "terse" in the sense that it only outputs the given address as a hex-value

Reply:
Done.